src/lib.rs

```diff
@@ -1217,11 +1219,15 @@ pub mod pallet {
 			}
 			<VoteCount<T>>::mutate(|count| count.saturating_inc());
 			// todo: confirm dispute fee handling with Tellor
-			// require(
-			// 	token.transferFrom(msg.sender, address(this), _disputeFee),
-			// 	"Fee must be paid"
-			// ); // This is the dispute fee. Returned if dispute passes
 			let dispute_fee = vote.fee;
+			// should we check for funds availability?
+			let pallet_id = T::PalletId::get();
+			T::Token::transfer(
+				&dispute_initiator,
+				&pallet_id.into_sub_account_truncating(b"dispute"),
```

Comment:
Would moving b"dispute" to constants.rs make sense, as the same value is being used across files?